napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -708,9 +720,7 @@ def __init__(self, parent=None):
         self.refresh_state = RefreshState.DONE
         self.already_installed = set()
 
-        installer_type = "pip"
-        if running_as_constructor_app():
-            installer_type = "mamba" if os.name != "nt" else "conda"
+        installer_type = "mamba" if running_as_constructor_app() else "pip"
```

Comment:
we'll never be conda then?

Reply:
We are bundling `mamba` in the installer too, so we are safe to assume it will be there.

Reply:
maybe remove the conda variant from line 59 if it's a value that will never be reached ?


```
InstallerTypes = Literal['pip', 'conda', 'mamba']
```